wslpy/exec.py

```diff
@@ -3,6 +3,42 @@
 This is the execution class to execute commands from different windows executables
 """
 import subprocess
+import os
+from wslpy.__init__ import isWSL
+import sys
+
+
+def __isInteropEnabled__():
+    cat_wslInterop = subprocess.getoutput(
+        "cat /proc/sys/fs/binfmt_misc/WSLInterop | grep '.*abled'")
+    return cat_wslInterop

```

Comment:
use `return cat_wslInterop == 'enabled'` here and use `return __isInteropEnabled__()` on line 25

Reply:
affirmative

Reply:
done